starter-code/app.js

```diff
@@ -4,8 +4,16 @@ const logger         = require("morgan");
 const cookieParser   = require("cookie-parser");
 const bodyParser     = require("body-parser");
 const mongoose       = require("mongoose");
+const bcrypt     	 = require("bcrypt");
```

Comment:
keep your files clean of variables/constants not used ... `bcrypt` is not used in this file